pay-queue/src/pay_queue/services/cgi_reconciliations.py

```diff
@@ -222,35 +222,53 @@ def _build_jv_details(line, receipt_number) -> JVDetailsFeedback:
         invoice_return_message=line[319:469],
         receipt_number=receipt_number,
     )
-    # Check if this is a partial refund by looking for "-PR" suffix
     flowthrough = details.flowthrough
-    details.is_partial_refund = "-PR" in flowthrough
+    details.is_partial_refund = False
```

Comment:
update logic because I found some conflicts while testing partial refund partner disbursement and EJV disbursement.
here is a summary to all scenarios:
1. Regular invoice ("1111")
Straightforward invoice ID parsing
Uses standard INVOICE link type
2. Partner disbursement ("1111-2222")
Format: INVOICE_ID-PARTNER_ID
Sets partner_disbursement from ID 2222
Uses standard INVOICE link type
3. Partner partial refund ("1111-2222-PR")
Format: INVOICE_ID-PARTNER_ID-PR
Gets partial_refund_id from partner_disbursement.target_id
Uses PARTIAL_REFUND link type with partial_refund_id
4. Direct partial refund ("1111-PR-3333")
Format: INVOICE_ID-PR-REFUND_ID
Directly extracts partial_refund_id (3333)
Uses PARTIAL_REFUND link type with partial_refund_id

Reply:
Why are 3 -4 any different?